src/components/Paginator/Paginator.tsx

```diff
@@ -35,6 +38,10 @@ type IPaginatorSingleSelectProps = Partial<ISingleSelectProps>;
 
 type ShowTotalObjects = (count: number) => string;
 
+interface IExtendedTextFieldProps extends Omit<ITextFieldProps, 'value'> {
+	value?: string | number;
```

Comment:
is this being done just to allow `number` as a type on value?

Reply:
Updated to only allow string as an optional value